starters/express-apollo-prisma/README.md

```diff
@@ -0,0 +1,190 @@
+- [ESLint](https://eslint.org/) - Code linting
+- [Prettier](https://prettier.io/) - Code formatting
+
+## Installation
+
+### CLI (Recommended)
+
+```bash
+npm create @this-dot/starter --kit express-apollo-prisma
+```
+
+or
+
+```bash
+yarn create @this-dot/starter --kit express-apollo-prisma
+```
+
+- Follow the prompts to select the `express-apollo-prisma` starter kit and name your new project.
+- `cd` into your project directory and run `npm install`.
+- Make sure you have docker & docker-compose installed on your machine
+- Create a `.env` file and copy the contents of `.env.example` into it.
+- Run `npm run docker:mount` to start the database and the Redis instances
+- Run `npm run start` to start the development server.
+- Open your browser to `http://localhost:4001` to see the API documentation with the existing endpoints.
+
+### Manual
+
+```bash
+git clone https://github.com/thisdot/starter.dev.git
+```
+
+- Copy and rename the `starters/express-apollo-prisma` directory to the name of your new project.
+- Make sure you have docker & docker-compose installed on your machine
+- `cd` into your project directory and run `npm install`.
+- Make sure you have docker & docker-compose installed on your machine
+- Create a `.env` file and copy the contents of `.env.example` into it.
+- Run `npm run docker:mount` to start the database and the Redis instances
+- Run `npm run start` to start the development server.
+- Open your browser to `http://localhost:4001` to see the API documentation with the existing endpoints.
+
+## Commands
+
+- `npm run docker:mount` - Starts up a Mysql database and two Redis instances for caching
+- `npm run docker:unmount` - Stops the running database and Redis docker containers.
+<!-- TODO - `npm run db:seed` - Allows you to seed the database (See the Seeding section) -->
+- `npm run dev` - Starts the development server (Needs a running infrastructure first)
+- `npm run build` - Builds the app.
+- `npm start` - Starts the built app. (Needs a running infrastructure first)
+- `npm test` - Runs the unit tests.
+- `npm run lint` - Runs ESLint on the project.
+- `npm run format` - Formats code for the entire project.
+- `npm prisma:format` - Updates your database using migrations during development and creates the database if it does not exist.
+- `npm prisma:migrate:reset` - Deletes and recreates the database, or performs a 'soft reset' by removing all data, tables, indexes, and other artifacts.
+- `npm prisma:migrate:dev` - updates your database using migrations during development and creates the database if it does not exist.
+- `npm run prisma:generate` - Generates the API schema types into the `src/interfaces/schema.ts` file
+- `npm run prisma:deploy` - Applies all pending migrations, and creates the database if it does not exist. Primarily used in non-development environments.
+- `npm run docker:build` - Builds, (re)creates, starts, and attaches to containers to a service in the background.
```

Comment:
Having another look at the commands, it looks like we need to go over all the `docker` commands. I'll create a ticket to cover the work.

Reply:
See #930